lib/livebook_web/live/session_live/indicators_component.ex

```diff
@@ -105,13 +106,29 @@ defmodule LivebookWeb.SessionLive.IndicatorsComponent do
 
   defp persistence_indicator(%{dirty: false} = assigns) do
     ~H"""
-    <span class="tooltip left" data-tooltip="Notebook saved">
+    <span
+      class="tooltip left"
+      data-tooltip={
+        case @persistence_warnings do
+          [] ->
+            "Notebook saved"
+
+          warnings ->
+            "Notebook saved with warnings:\n" <> Enum.map_join(warnings, "\n", &("- " <> &1))
```

Comment:
Should each of those be in their own line? That doesn't seem to be the case in the screenshot below.

Reply:
It is, it's just that the message includes a dash.